metadata/health/health_small_statistical_areas_2020.md

```diff
@@ -0,0 +1,73 @@
+# Title
+
+Utah Health Small Statistical Areas 2020
+
+## ID
+
+34f8331f-c0d9-48d6-83d9-adcd0ae28302
+
+## Brief Summary
+
+Polygon dataset used by the Utah Department of Health and Human Services for statistical analysis and reporting.
+
+## Summary
+
+This dataset contains polygons representing small statistical areas as defined by the [Utah Department of Health and Human Services](https://dhhs.utah.gov/) (DHHS) for the year 2020. Previous versions of the Small Health Statistical Areas datasets are available on our [ArcGIS Online Shelf](https://utah.maps.arcgis.com/home/group.html?id=8765687b7b0545668fff02d1b750f7a9#overview) page.
+
+## Description
+
+### What is the dataset?
+
+Small statistical areas are zones defined by DHHS that have an approximate population of 8,000 to 86,000 people. These zones were first developed in 1997 and are used to aggregate data for public health research. You can view a variety of studies by small area on the [DHSS data catalog](https://opendata.utah.gov/browse?q=small+area&sortBy=relevance&pageSize=20).
+
+### What is the purpose of the dataset?
+
+DHHS uses this dataset as a framework for more meaningful, focused statistical analysis. This dataset has been made available to the wider public for general research and cartographic purposes.
+
+### What does the dataset represent?
+
+Each feature in this dataset represents the geographic extent of a small statistical area.
+
+<!--- This dataset has a field called 'SmallAreaNumber' which I initially thought was an ID number for each small area, but some of the numbers in the field are decimals, which makes me think that maybe it represents something else. Do we know what this number indicates? --->
+
+### How was the dataset created?
+
+The Office of Public Health Assessment at DHHS developed this layer using the [Utah Local Health District and Small Area Designation Algorithm](https://ibis.utah.gov/ibisph-view/pdf/resource/Algorithm.pdf). They defined small areas based on ZIP Codes, population, local health district and county boundaries, income levels, community political boundaries, and input from local community representatives. The resulting 99 small areas may not always have a contiguous boundary, in which case the area is represented with multiple polygons. These data are maintained by DHHS in collaboration with UGRC.
```

Comment:
I think the boundaries were defined with only zip codes and county boundaries. Political boundaries or input community input aren't used to define these boundaries.